cirq-ionq/cirq_ionq/ionq_devices.py

```diff
@@ -36,6 +36,19 @@ class IonQAPIDevice(cirq.Device):
         * `cirq.MeasurementGate`
     """
 
+    gateset = cirq.Gateset(
+        cirq.H,
+        cirq.CNOT,
+        cirq.SWAP,
+        cirq.XPowGate,
+        cirq.YPowGate,
+        cirq.ZPowGate,
+        cirq.XXPowGate,
+        cirq.YYPowGate,
+        cirq.ZZPowGate,
+        cirq.MeasurementGate,
```

Comment:
Does this work for cirq.SWAP**3?  Reading the gateset code I think it does, but wanted to check.

Reply:
Yes, it'll work for `cirq.SWAP ** 3` because `cirq.SWAP ** 3 == cirq.SWAP` and the gateset will use value equality by default for gate instances.